website/public/js/controllers/tasksCtrl.js

```diff
@@ -220,8 +220,8 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
     $scope.armoire = Content.armoire;
 
     $scope.buy = function(item) {
-      User.user.ops.buy({params:{key:item.key}});
-      playRewardSound(item);
+        playRewardSound(item);
+        User.user.ops.buy({params:{key:item.key}});
```

Comment:
Whoops! Those were inserted by my IDE. Slipped my notice. I'll fix that tonight.